services/alerts/src/AlertsProvider.tsx

```diff
@@ -5,15 +5,15 @@ import { makeAlertsManager } from './makeAlertsManager'
 import type { Alert, AlertsManager } from './types'
 
 export const AlertsProvider = ({
-    plugin,
-    parentAlertsAdd,
-    showAlertsInPlugin,
     children,
+    plugin = false,
+    parentAlertsAdd = undefined,
+    showAlertsInPlugin = false,
 }: {
-    plugin: boolean
-    parentAlertsAdd: any
-    showAlertsInPlugin: boolean
     children: React.ReactNode
+    plugin?: boolean
```

Comment:
`plugin` should actually always be defined as long as you are using an up-to-date version of app-platform (https://github.com/dhis2/app-platform/blob/master/shell/src/App.js#L43). I'm not sure if there's a scenario where we want to mark it as an optional in case you're trying to resolve to a higher-version of app-runtime while using an older version of app-platform?